packages/web-components/fast-components/package.json

```diff
@@ -33,7 +33,8 @@
     "start": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "generate-default-palettes": "node -r esm build/generate-default-palettes.js",
-    "test": "yarn test-chrome:verbose && yarn doc:ci",
+    "generate-open-ui-definition": "tsc -p ./tsconfig.definitions.json && node -r esm build/generate-open-ui-definition",
+    "test": "yarn run build && yarn test-chrome:verbose && yarn doc:ci",
```

Comment:
This is going to cause ci to run build twice

Reply:
Ah, I can remove, the goal was to generate the open ui definitions and run the validator against them